dimagi/pages/templates/svg/tookits/icons/managingdevices_scale.html

```diff
@@ -0,0 +1,36 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 80 80" width="110" height="110">
```

Comment:
interesting,  the `80 80` should be `110 110` in this case as this also corresponds to the height and width, no? Or are you scaling up the icon from `80x80`

Reply:
No, in this case `80 80` is not corresponding to height and width. According to mockup the icons should be little bit larger than the actual size of icons existing in website, so that's why here `80 80` is scaling up the icon. If we are keeping both same, the UI and icons are getting disturbed.
I am adding two screenshots which depicts how icon is looking after keeping `80 80` and `110 110` for both viewBox and height, width.

![screenshot from 2019-01-16 11-06-33](https://user-images.githubusercontent.com/15342814/51228593-2ee7e080-197f-11e9-950e-1ca609001a1a.png)
![screenshot from 2019-01-16 11-07-12](https://user-images.githubusercontent.com/15342814/51228756-c2211600-197f-11e9-83d2-934e1003e5fb.png)